packages/react-magma-dom/src/components/CharacterCounter/CharacterCounter.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { axe } from '../../../axe-helper';
+import { CharacterCounter } from '.';
+import { render } from '@testing-library/react';
+
+const testId = 'test-id';
+
+describe('CharacterCounter', () => {
+  it('should find element by testId', () => {
+    const { getByTestId } = render(
+      <CharacterCounter maxLength={231} testId={testId} />
+    );
+
+    expect(getByTestId(testId)).toBeInTheDocument();
+  });
+
+  it('Does not violate accessibility standards', () => {
+    const { container } = render(<CharacterCounter maxLength={22} />);
+
+    return axe(container.innerHTML).then(result => {
+      return expect(result).toHaveNoViolations();
+    });
+  });
+});
```

Comment:
As far as CharacterCounter unit tests, we should write some that directly test this component. So for example, some of the tests that you wrote that belong here would be:

```
it('Shows the default label of "characters allowed" if maxLength is 0', () => {
        const { getByText } = render(
          <I18nContext.Provider
            value={{
              ...defaultI18n,
            }}
          >
            <CharacterCounter maxLength={0} />
          </I18nContext.Provider>
        );

        expect(getByText('0 ' + charactersAllowed)).toBeInTheDocument();
      });

      it('Shows the default label of "character allowed" if maxLength is 1', () => {
        const { getByText } = render(
          <I18nContext.Provider
            value={{
              ...defaultI18n,
            }}
          >
            <CharacterCounter maxLength={1} />
          </I18nContext.Provider>
        );

        expect(getByText('1 ' + characterAllowed)).toBeInTheDocument();
      });
      it('Shows the default label of "characters allowed" if maxLength is greater than 1', () => {
        const { getByText } = render(
          <I18nContext.Provider
            value={{
              ...defaultI18n,
            }}
          >
            <CharacterCounter maxLength={2} />
          </I18nContext.Provider>
        );

        expect(getByText('2 ' + charactersAllowed)).toBeInTheDocument();
      });
    });
```

Notice above that the change I made was call `<CharacterCounter maxLength={1} />` **instead of** `<Input maxLength={1} />`. In addition, we want to add tests that directly test the other CharacterCounter prop of `inputLength`. 

Then, any test that checks things like `onChange` or any Input prop should live under the Input tests files. In addition, we want to add tests similar to the Input ones to Textarea to test that the CharacterCounter works within that component as well. 

Let me know if you need further clarification.